src/main/resources/templates/macros/json/embeddings.vm

```diff
@@ -79,7 +79,7 @@
   <div class="embedding indention">
     #set($embeddingId = $counter.next())
     <div data-toggle="collapse" data-target="#embedding-$embeddingId" class="collapsable-control">
```

Comment:
this `div` can be now moved to separate template to avoid code repeating but this is not the scope of this pull request